test/presenters/return-requirements/returns-cycle.presenter.test.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Thing under test
+const ReturnsCyclePresenter = require('../../../app/presenters/return-requirements/returns-cycle.presenter.js')
+
+describe('Returns Cycle presenter', () => {
+  let session
```

Comment:
The let session block can go down a describe block. It's good practise to only declare the variables when you need them. Since your next describe block is 'when provided with a populated session' it would be good to declare the session there (since it is within the context).